client/questions_answers/components/HelpfulReport.jsx

```diff
@@ -6,17 +6,22 @@ const HelpfulReport = (props) => {
   let helpfulIds = JSON.parse(localStorage.getItem(props.content_type === 'question' ? 'helpfulQuestions' : 'helpfulAnswers'));
 
   if (props.reported) {
-    renderReportedLink = (<span> Reported </span>);
+    renderReportedLink = (<span> <i>Reported</i> | </span>);
   } else {
-    renderReportedLink = (<button onClick={props.handleReportButtonClick}> Report </button>);
+    renderReportedLink = ( <span>
+      <button
+        className="qa-button-link" onClick={props.handleReportButtonClick}> Report </button>
+    </span>
+    );
   }
 
   if (helpfulIds.includes(props.content_id)) {
-    renderHelpfulLink = (<span> {`Helpful? Yes ${props.helpfulness}`}</span>);
+    renderHelpfulLink = (<span> {'Helpful?'} <i>Yes</i> {`(${props.helpfulness}) | `}</span>);
```

Comment:
Do you still need JSX if you're just rendering a string?

Reply:
Ha yeah. There were a couple cases where I needed a space before or after text and that's the way I got it to work. Probably not the best way to do that. I guess I just decided to keep going with it! I'll figure out a better solution and refactor.